lib/ueberauth.ex

```diff
@@ -193,8 +193,14 @@ defmodule Ueberauth do
 
   @doc false
   def init(opts \\ []) do
+    config_opts = Application.get_env(:ueberauth, Ueberauth)
```

Comment:
You could do `Application.get_env!(:ueberauth, Ueberauth)` and you don't need to add the error message at that point if I am not mistaken.

`get_env!` is meant to be used when you have this use case.

Reply:
Is `Application.get_env!/3` new to Elixir 1.8? I'm not seeing it in the [docs for Elixir 1.7.4](https://hexdocs.pm/elixir/Application.html). I tried compiling against it anyways in case the docs were wrong but my compiler can't find that function.

Reply:
Sorry I meant to say `fetch_env/2`, my bad